src/bot/BotController.ts

```diff
@@ -51,4 +56,6 @@ export class BotController {
   };
 }
 
+export const bot = new BotController();
```

Comment:
We initialize the bot here now so we can import it into other functions and access the client if necessary.